internal/subshell/subshell.go

```diff
@@ -140,6 +142,45 @@ func New(cfg sscommon.Configurable) SubShell {
 	return subs
 }
 
+func availableShells() []SubShell {
+	var shells []SubShell
+	for _, shell := range supportedShells {
+		rcFile, err := shell.RcFile()
+		if err != nil {
+			logging.Error("Could not determine rc file for shell %s: %v", shell.Shell(), err)
+			continue
+		}
+
+		if !fileutils.FileExists(rcFile) {
```

Comment:
```suggestion
		if !fileutils.TargetExists(rcFile) {
```

This could reasonably be a symlink.